js/tetris.js

```diff
@@ -4,20 +4,41 @@ var lose;
 var interval;
 var current; // current moving shape
 var currentX, currentY; // position of current shape
+
+var score = 0;
+var pause = false;
+
+var button = document.getElementById("newgame");
+button.addEventListener ("click", newGame);
```

Comment:
Please keep code formatting according to the rest of the codebase: Single quotes, space after `,`, no space before `(`, and so on